### PR TITLE
[#10725] Show student name more prominently

### LINKED ISSUE #10725: Student submitting responses: show own name more prominently
When students submit responses, the student name (see below) can be shown more prominently (bigger letters, bold, different color etc.)
image

This is for students to self-detecting when submitting responses using other people's links. Despite our warning of not to do so, sometimes students share their submission links.

### PR DESCRIPTION
**Pull request by ankita2210**
Fixes #10725 

**Outline of Solution**

- Added bold tag and mark tag to make the student name appear more prominent while submitting responses

### CODE REVIEW

**Comment by hcwong** [comment]
Hello, this looks fine to me, but could I trouble you to add in a before and after screenshot of the page. thanks! lmk once its done and ill approve it

File: src/web/app/pages-session/session-submission-page/session-submission-page.component.html [hunk 1]
```diff
@@ -31,11 +31,11 @@ <h4>Previewing Session as
...
     <div class="alert alert-primary" role="alert">
       <div *ngIf="loggedInUser">
-        You are submitting feedback as {{ personName }}. If you wish to link your Google account ({{ loggedInUser }}) with this user,
+        You are submitting feedback as <mark><b>{{ personName }}</b></mark>. If you wish to link your Google account ({{ loggedInUser }}) with this user,
         <a href="#" (click)="joinCourseForUnregisteredStudent(); $event.preventDefault()">click here</a>.
       </div>
       <div *ngIf="!loggedInUser">
-        You are submitting feedback as {{ personName }}. You may submit feedback for sessions that are currently open and view results without logging in.
+        You are submitting feedback as <mark><b>{{ personName }}</b></mark>. You may submit feedback for sessions that are currently open and view results without logging in.
         To access other features you need to <a href="#" (click)="joinCourseForUnregisteredStudent(); $event.preventDefault()">login using a Google account</a> (recommended).
```
Comment (damithc) [inline src/web/app/pages-session/session-submission-page/session-submission-page.component.html:39]: @ankita2210 can you also change `(recommended)` to `(optional)`?
@hcwong I would like to squeeze in the above change to this PR as well, FYI

**Comment by ankita2210** [comment]
Will do the needful in a couple of hours

**Comment by ankita2210** [comment]
> Hello, this looks fine to me, but could I trouble you to add in a before and after screenshot of the page. thanks! lmk once its done and ill approve it

image

**Review by hcwong — APPROVED** [review]
lgtm

**Review by madanalogy — APPROVED** [review]
LGTM! Thanks for your contribution.

**Pull request merged**